mongo/datadog_checks/mongo/collectors/replica.py

```diff
@@ -131,4 +131,15 @@ def collect(self, client):
             replset_name = status['set']
             self._report_replica_set_states(status['members'], replset_name)
 
+            # Compute a lag time
+            for member in status.get('members', []):
+                if 'optimeDate' in primary and 'optimeDate' in member:
+                    lag = primary['optimeDate'] - member['optimeDate']
+                    tags = self.check.base_tags + [
+                        'member:{}'.format(member.get('name', 'unknown')),
+                        'replset_name:{}'.format(replset_name),
+                        'replset_state:{}'.format(get_state_name(member.get('state')).lower()),
+                    ]
+                    self.gauge('mongodb.replset.lag_from_primary', lag.total_seconds(), tags)
```

Comment:
WDYTA `mongodb.replset.primary_lag`? `mongodb.replset.primary_optime_lag`?

`lag_from_primary` does feel a bit "inverted" (usually 'y of the x' becomes `something.x.y` rather than `something.y.x`), but it's acceptable too.